clients/cmd/logstash/Dockerfile

```diff
@@ -1,24 +1,24 @@
-FROM logstash:7.9.3
-
-USER logstash
-ENV PATH /usr/share/logstash/vendor/jruby/bin:/usr/share/logstash/vendor/bundle/jruby/2.5.0/bin:$PATH
-ENV LOGSTASH_PATH="/usr/share/logstash"
-ENV GEM_PATH /usr/share/logstash/vendor/bundle/jruby/2.5.0
-ENV GEM_HOME /usr/share/logstash/vendor/bundle/jruby/2.5.0
-
-RUN gem install bundler:2.1.4
-
-COPY --chown=logstash:logstash ./clients/cmd/logstash/ /home/logstash/
```

Comment:
is it fine to remove this ? Do you think it's not used ?

Reply:
please do not remove it.